indico/modules/events/abstracts/templates/forms/rule_list_widget.html

```diff
@@ -0,0 +1,31 @@
+{% extends 'forms/base_widget.html' %}
+{% from 'message_box.html' import message_box %}
+
+{% block html %}
+
+    <input type="hidden" id="{{ field.id }}" name="{{ field.name }}" value="{{ field._value() }}">
+    <div class="js-rule-list-widget">
+        <ul class="rule-list">
+        </ul>
+        {% call message_box('info', classes='no-rules-placeholder') %}
+            {% trans %}No rules have been added{% endtrans%}
+        {% endcall %}
+        <div class="toolbar">
+            <div class="group right">
+                <button class="i-button icon-plus highlight js-add-new-rule">
+                    {% trans %}Add new rule{% endtrans %}
+                </button>
+            </div>
+        </div>
+    </div>
+{% endblock %}
+
+{% block javascript %}
+    <script>
+        $('#{{ field.id }}').rulelistwidget({
+            containerElement: $('.js-rule-list-widget'),
```

Comment:
unlikely to happen, but this breaks if you have more than one on the same page. I'd rather use something relative to `#{{ field.id }}`